tests/unit/test_prefixbuf.cpp

```diff
@@ -0,0 +1,134 @@
+#include "../gtest.h"
+
+#include <cstring>
+#include <iomanip>
+#include <string>
+#include <sstream>
+
+#include <util/prefixbuf.hpp>
+
+using namespace arb::util;
+
```

Comment:
Comments above each test case to state what is being tested, because the test is the documentation.